third_party/silabs/efr32_sdk.gni

```diff
@@ -82,6 +82,9 @@ declare_args() {
   chip_enable_multi_ota_requestor = false
   chip_enable_multi_ota_encryption = false
   chip_enable_ota_custom_tlv_testing = false
+
+  # Factory Provision storage
+  use_flash_storage = wifi_soc
```

Comment:
we should add `asserts` to make sure the flash/non flash makes sense with the crypto flavor.

Reply:
Technically they could/can be independent